Features/Stories/Fieldtypes/EZP-2477-AuthorsFieldType.feature

```diff
@@ -0,0 +1,97 @@
+Feature: Test the validations done on Author fieldtype from Editorial Interface
+
+    @javascript
+    Scenario: Message when using empty author field
+        Given I go to PlatformUI app with username "admin" and password "publish"
```

Comment:
+1 (more or less, but we did try something like this last year, lets make sure it does not grow into the same abstract level)